docs/design/design.md

```diff
@@ -43,7 +43,7 @@ Agent is running constantly on the edge device and responsible for:
 
 Agent is running in two processes:
  - [yggdrasil](https://github.com/jakub-dzon/yggdrasil/) - gateway service responsible for communication with the control plane; dispatches messages to and from the control plane from/to workers (extensions);
- - [device-worker](https://github.com/jakub-dzon/k4e-device-worker) - yggdrasil worker responsible for performing all the logic listed above.
+ - [device-worker](https://github.com/flotta/flotta-device-worker) - yggdrasil worker responsible for performing all the logic listed above.
```

Comment:
I know that device-worker is still not present, but I think this should be `https://github.com/project-flotta/flotta-device-worker`

Reply:
Yes! Thank you!